install.sh

```diff
@@ -11,8 +11,20 @@ if [ "$1" == "development" ]; then
 fi
 
 cd $ROOTPATH
-curl -fSL "https://s3.amazonaws.com/rocketchatbuild/rocket.chat-develop.tgz" -o rocket.chat.tgz
-tar zxf rocket.chat.tgz  &&  rm rocket.chat.tgz
+gpg --keyserver ha.pool.sks-keyservers.net --recv-keys 0E163286C20D07B9787EBE9FD7F9D0414FD08104
+
+curl -SLf "https://releases.rocket.chat/latest/download/" -o rocket.chat.tgz
```

Comment:
the `develop` release is actually at `https://releases.rocket.chat/develop/download/` .. or did you change to `latest` on purpose?

Reply:
Since it's a script never used, and wouldn't be updated between master and develop I figured it better to have anyone using it to actually install latest stable